variables.tf

```diff
@@ -889,7 +889,7 @@ variable "clusters" {
             entitlement          = optional(string)      # entitlement option for openshift
             secondary_storage    = optional(string)      # Secondary storage type
             boot_volume_crk_name = optional(string)      # Boot volume encryption key name
-            operating_system     = optional(string)      # The operating system of the workers in the default worker pool. If no value is specified, the current default version OS will be used. See https://cloud.ibm.com/docs/openshift?topic=openshift-openshift_versions#openshift_versions_available .
+            operating_system     = string                # The operating system of the workers in the default worker pool. See https://cloud.ibm.com/docs/openshift?topic=openshift-openshift_versions#openshift_versions_available .
```

Comment:
I'm  worried about some teams that call this root level directly, and we are changing this from optional to required. Would it make sense to keep this as optional but with default of REDHAT_8_64? `optional(string, "REDHAT_8_64")`